daisy_workflows/image_build/enterprise_linux/kickstart/ol7-post.cfg

```diff
@@ -0,0 +1,122 @@
+# ol7-post.cfg
+# Google Compute Engine kickstart config for Oracle Linux 7
+# See ks_helpers.py for details on how the options, packages, pre and post files fit
+# together.
+
+# 1
+%post
+
+# Oracle public Yum repo config.
+curl -sSL 'http://yum.oracle.com/public-yum-ol7.repo' > /etc/yum.repos.d/public-yum-ol7.repo
+
+# Enable addons repo for python-boto.
+sed -i'' -e '/ol7_addons/,/^$/{ s/enabled=0/enabled=1/ }' \
+	/etc/yum.repos.d/public-yum-ol7.repo
+
+# Delete the dummy user account.
+userdel -r gce
+
+%end
+
+# 2
+%post
+# Configure the network for GCE.
+
+# Given that GCE users typically control the firewall at the network API level, we
+# want to leave the standard Linux firewall setup enabled but all-open.
+firewall-offline-cmd --set-default-zone=trusted
+
+# Set the MTU
+cat >>/etc/sysconfig/network-scripts/ifcfg-eth0 <<EOL
+MTU=1460
+EOL
+
+# Set the dhclient retry interval to 10 seconds instead of 5 minutes.
+# NetworkManager will merge custom options into the config it generates.
+cat >>/etc/dhclient.conf <<EOL
+# Set the dhclient retry interval to 10 seconds instead of 5 minutes.
+retry 10;
+EOL
+
+# Set dhclient to be persistent instead of oneshot.
+# This applies to EL 6/CentOS 6/OL 6 and EL 7/CentOS 7/OL 7 if NetworkManager is disabled.
+cat >>/etc/sysconfig/network-scripts/ifcfg-eth0 <<EOL
+PERSISTENT_DHCLIENT="y"
+EOL
+
+# Disable IPv6.
+cat >>/etc/sysctl.d/70-disable-ipv6.conf <<EOL
+net.ipv6.conf.all.disable_ipv6 = 1
+net.ipv6.conf.lo.disable_ipv6 = 0
+EOL
+
+%end
+
+# 3
+%post
+# Configure the system to run in GCE.
+
+# Install GCE supplied disk expansion package.
+yum install -y gce-disk-expand
+
+# Install packages needed by GCE.
+yum install -y google-compute-engine google-config google-compute-engine-init
```

Comment:
Do you know if python-boto is available? That was one of the issues I had in the past.

Reply:
Yes, it's in the ol7_addons repo which is enabled above.